### DISCUSSION TITLE
[Beginner help] Mockup browser not working, everything else works

### DISCUSSION THREAD

**Discussion by 42willow** [saadeghi/daisyui#2190] [Q&A] [answered]
So far I've tried a few components, but only half seem to work for some reason 😕 
### Works:
- **ALL actions**
- **ALL data inputs**
- All mockups but browser
- Steps
- Chat Bubbles
- Table
### Doesn't work: (everything labelled new)
- Browser mockup
- Loading
- Accordion
- Join (group items)

I'm probably missing something blindingly obvious, the browser mock-up looks like this, and the loading spinners don't appear.

## App.svelte
```svelte
<div class="mockup-browser border bg-base-300">
    <div class="mockup-browser-toolbar">
      <div class="input">https://daisyui.com</div>
    </div>
    <div class="flex justify-center px-4 py-16 bg-base-200">Hello!</div>
</div>

<span class="daisy-loading daisy-loading-spinner daisy-loading-xs"></span>
```
## App.css
```css
@tailwind base;
@tailwind components;
@tailwind utilities;
```
## main.js
```js
import './app.css';
import App from './App.svelte';

const app = new App({
  target: document.getElementById('app'),
});

export default app;
```
Here's the GitHub repo: https://github.com/42willow/nocrastinate
![image](https://github.com/saadeghi/daisyui/assets/70784752/d82db2d9-300a-47ed-a9fb-d36cfa92c37b)

**Comment by 42willow** [comment]
I noticed a common pattern that the new components weren't working and updated daisyui o.o

**Comment by saadeghi** [comment, marked as answer]
I think you're using daisyUI 2. Update to the latest version.
